### DISCUSSION TITLE
如何实现参考文献按作者排序

### DISCUSSION THREAD

**Discussion by xuchenqi91-op** [sjtug/SJTUThesis#798] [提问]
我写的是数学本科毕业论文，老师要求参考文献按作者排序，而模板默认为按文中顺序排序。
对于普通的latex文档，只需在导言区加上
`\usepackage[backend=biber,style=gb7714-2015ay,maxbibnames=5,minbibnames=5]{biblatex}`
即可。但对于模板文件，尝试加上上述语句后报错。请问应该如何操作？

**Comment by skyzh** [comment]
在 setup.tex 里修改即可，里面有类似的控制语句。

**Reply by xuchenqi91-op** [reply]
嗯嗯已解决，将原代码改成这个就行
`\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase,gbpub=false,gbpunctin=false,sorting=gb7714-2015,gblanorder=chineseahead,sortlocale=zh__pinyin]{biblatex}`
